Formula/dura.rb

```diff
@@ -27,6 +27,14 @@ def install
     system "cargo", "install", *std_cargo_args
   end
 
+  service do
+    run [opt_bin/"dura", "serve"]
+    keep_alive true
+    error_log_path var/"log/dura.log"
+    log_path var/"log/dura.log"
```

Comment:
Should this be different from the error path, since it seems to produce JSON?

Reply:
We could separate them. Is there a convention we should use? Looking at other formulas, seems like for errors, we could do `dura.err`, `dura.err.log`, `dura-error.log`, or `dura/error.log`.

Reply:
cc @tkellogg

Reply:
@tkellogg's example plist uses `dura.log.json`. I guess we can use that.

Reply:
Updated both filenames to match the example plist (`dura.log.json` and `dura.stderr.log`).